scripts-dev/make_full_schema.sh

```diff
@@ -293,4 +299,12 @@ pg_dump --format=plain --data-only --inserts --no-tablespaces --no-acl --no-owne
 pg_dump --format=plain --schema-only         --no-tablespaces --no-acl --no-owner "$POSTGRES_STATE_DB_NAME"  | cleanup_pg_schema  > "$OUTPUT_DIR/state/full_schemas/$SCHEMA_NUMBER/full.sql.postgres"
 pg_dump --format=plain --data-only --inserts --no-tablespaces --no-acl --no-owner "$POSTGRES_STATE_DB_NAME"  | cleanup_pg_schema >> "$OUTPUT_DIR/state/full_schemas/$SCHEMA_NUMBER/full.sql.postgres"
 
+if [ "$OUTPUT_DIR" == *synapse/storage/schema ]; then
+  echo "Updating contrib/datagrip symlinks..."
+  ln -sf "synapse/storage/schema/common/full_schemas/$SCHEMA_NUMBER/full.sql.postgres" "contrib/datagrip/common.sql"
+  ln -sf "synapse/storage/schema/main/full_schemas/$SCHEMA_NUMBER/full.sql.postgres"   "contrib/datagrip/main.sql"
+  ln -sf "synapse/storage/schema/state/full_schemas/$SCHEMA_NUMBER/full.sql.postgres"  "contrib/datagrip/state.sql"
```

Comment:
I think this needs updating?

Reply:
Err, yes! Sorry, should have marked this as a draft again.

Reply:
Err, actually I think this is correct. (`ln -s TARGET SYMLINK_LOCATION`)

Reply:
Oh no, it needs some `../../`

Reply:
Arg, I always flip `ln` arguments around. 😢

Reply:
I tested the latest version in d383945e46 and this seemed to work for me.